bin/node/cli/src/chain_spec.rs

```diff
@@ -770,6 +774,8 @@ pub fn pangolin_local_testnet_config() -> PangolinChainSpec {
 				phantom: PhantomData::<pangolin_runtime::TechnicalMembershipInstance>,
 				members: collective_members.clone(),
 			},
+			pallet_treasury: Default::default(),
```

Comment:
Why not `pallet_treasury_Instance1`？ like `pallet_collective_Instance1`, `pallet_collective_Instance2`.

Reply:
https://github.com/darwinia-network/darwinia-common/pull/759/files#diff-b33acff65d93ddb65c199e2c906bbc3c862ff6f287dd364a34c4c6e6bfcf46f1R17